decisions/0009-schema-refactor.md

```diff
@@ -0,0 +1,89 @@
+---
+status: proposed {proposed | rejected | accepted | deprecated | … | superseded by [ADR-0005](0005-example.md)}
+date: 2023-12-19{YYYY-MM-DD when the decision was last updated}
+deciders: Chris Mungall, Lee Ann {list everyone involved in the decision}
+consulted: Anastasiya Prymolenna, Brynn Zalmanek, James Tessmer, Montana Smith, Sam Purvine, Yuri Corilo, Mark Miller, Michael Thorton, Alicia Clum, Mark Miller    {list everyone whose opinions are sought (typically subject-matter experts); and with whom there is a two-way communication}
+informed: Emiley Eloe-Fadrosh, Shreyas Cholia, Eric Cavanna {list everyone who is kept up-to-date on progress; and with whom there is a one-way communication}
+---
+# Schema Refactoring: Monterey//Berkeley
+
+## Decisions Made
+
+### Class:PlannedProcess
+* `Class:PlannedProcess` will be created as a root class 
+* All "Activity" classes are now of `Class:PlannedProcess`, and the word "Activity" will be removed
```

Comment:
There is still a grouping layer. So WorkflowExecutionActivity became WorkflowExecution and inherits from PlannedProcess instead of Activity.

Reply:
Possible missing word: By "are now of", did you mean "are now (somethings) of"; where "(somethings)" is, for example, "subclasses", "descendants", etc.?

Reply:
> I'm not sure why you think the word sometimes should be here?

I was using "(somethings)" as a placeholder/variable; for example, by "are now (somethings) of", I meant it might be "are now _children_ of", "are now _descendants_ of", "are now _subclasses_ of", etc.—rather than "are now of". I didn't know what that missing word was, though.

Here's what I propose (you can replace "descendants" with whatever is accurate):

```diff
- * All "Activity" classes are now of `Class:PlannedProcess`, and the word "Activity" will be removed.
+ * All "Activity" classes are now descendants of `Class:PlannedProcess`, and the word "Activity" will be removed.
```

Reply:
@turbomam anything to add here. I would like to merge this Friday morning.